services/src/main/java/fr/ippon/tatami/config/CassandraConfiguration.java

```diff
@@ -42,121 +73,687 @@
-            addColumnFamily(cluster, ColumnFamilyKeys.TAG_FOLLOWERS_CF, 0);
-            addColumnFamily(cluster, ColumnFamilyKeys.GROUP_MEMBERS_CF, 0);
-            addColumnFamily(cluster, ColumnFamilyKeys.USER_GROUPS_CF, 0);
-            addColumnFamily(cluster, ColumnFamilyKeys.GROUP_CF, 0);
-            addColumnFamily(cluster, ColumnFamilyKeys.GROUP_DETAILS_CF, 0);
-            addColumnFamily(cluster, ColumnFamilyKeys.ATTACHMENT_CF, 0);
-            addColumnFamily(cluster, ColumnFamilyKeys.AVATAR_CF, 0);
-            addColumnFamily(cluster, ColumnFamilyKeys.DOMAIN_CONFIGURATION_CF, 0);
-            addColumnFamily(cluster, ColumnFamilyKeys.TATAMIBOT_CONFIGURATION_CF, 0);
-            addColumnFamily(cluster, ColumnFamilyKeys.APPLE_DEVICE_CF, 0);
-
-            addColumnFamilySortedbyUUID(cluster, ColumnFamilyKeys.TIMELINE_CF, 0);
-            addColumnFamilySortedbyUUID(cluster, ColumnFamilyKeys.TIMELINE_SHARES_CF, 0);
-            addColumnFamilySortedbyUUID(cluster, ColumnFamilyKeys.MENTIONLINE_CF, 0);
-            addColumnFamilySortedbyUUID(cluster, ColumnFamilyKeys.USERLINE_CF, 0);
-            addColumnFamilySortedbyUUID(cluster, ColumnFamilyKeys.USERLINE_SHARES_CF, 0);
-            addColumnFamilySortedbyUUID(cluster, ColumnFamilyKeys.FAVLINE_CF, 0);
-            addColumnFamilySortedbyUUID(cluster, ColumnFamilyKeys.TAGLINE_CF, 0);
-            addColumnFamilySortedbyUUID(cluster, ColumnFamilyKeys.TRENDS_CF, 0);
-            addColumnFamilySortedbyUUID(cluster, ColumnFamilyKeys.USER_TRENDS_CF, 0);
-            addColumnFamilySortedbyUUID(cluster, ColumnFamilyKeys.GROUPLINE_CF, 0);
-            addColumnFamilySortedbyUUID(cluster, ColumnFamilyKeys.USER_ATTACHMENT_CF, 0);
-            addColumnFamilySortedbyUUID(cluster, ColumnFamilyKeys.STATUS_ATTACHMENT_CF, 0);
-            addColumnFamilySortedbyUUID(cluster, ColumnFamilyKeys.DOMAINLINE_CF, 0);
-            addColumnFamilySortedbyUUID(cluster, ColumnFamilyKeys.DOMAIN_TATAMIBOT_CF, 0);
-
-            addColumnFamilyCounter(cluster, ColumnFamilyKeys.COUNTER_CF, 0);
-            addColumnFamilyCounter(cluster, ColumnFamilyKeys.TAG_COUNTER_CF, 0);
-            addColumnFamilyCounter(cluster, ColumnFamilyKeys.GROUP_COUNTER_CF, 0);
-            addColumnFamilyCounter(cluster, ColumnFamilyKeys.DAYLINE_CF, 0);
-
-            //Tatami Bot CF
-            addColumnFamily(cluster, ColumnFamilyKeys.TATAMIBOT_DUPLICATE_CF, 0);
-        }
-        return HFactory.createKeyspace(cassandraKeyspace, cluster, consistencyLevelPolicy);
+
+
+
+
+    @Inject
+    BeanFactory beanFactory;
+
+
+
+    /**
+     * Parse the load balancing policy.
+     */
+    public LoadBalancingPolicy parseLbPolicy(String loadBalancingPolicyString) throws InstantiationException,
+            IllegalAccessException, ClassNotFoundException, NoSuchMethodException, SecurityException,
+            IllegalArgumentException, InvocationTargetException {
+        String lb_regex = "([a-zA-Z]*Policy)(\\()(.*)(\\))";
```

Comment:
I'm not sure what you mean?
It tries to get the load balancing policy out together with the parameters for the load balancing